eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -56,20 +66,85 @@ def run_pipeline(self):
             try:
                 result = func(**self.backend_data) or {}
             except PearsonBaseError as pearson_error:
-                tasks = importlib.import_module("eox_nelp.pearson_vue.tasks")
-                tasks.rti_error_handler_task.delay(
-                    failed_step_pipeline=func.__name__,
-                    exception_dict=pearson_error.to_dict(),
-                    course_id=self.backend_data.get("course_id"),
-                    user_id=self.backend_data.get("user_id"),
-                )
+                self.handle_error(pearson_error, func.__name__)
                 break
 
             self.backend_data.update(result)
             if result.get("safely_pipeline_termination"):
                 self.backend_data["pipeline_index"] = len(pipeline) - 1
                 break
 
+    @abstractmethod
+    def get_pipeline(self):
+        """
+        Returns the pipeline, which is a list of functions to be executed.
+
+        Subclasses must override this method to provide the specific functions to be executed in the pipeline.
+
+        Returns:
+            list: A list of functions representing the pipeline.
+        """
+
+    @abstractmethod
+    def handle_error(self, exception, failed_step_pipeline):
+        """
+        Handles errors during pipeline execution.
+
+        Args:
+            exception (PearsonBaseError): The exception that was raised.
+            failed_step_pipeline (str): The name of the pipeline step where the error occurred.
+        """
+
+
+class ErrorRealTimeImportHandler(AbstractBackend):
+    """Class for managing validation error pipe  executing the pipeline for data validation."""
+
+    def handle_error(self, exception, failed_step_pipeline):
```

Comment:
I think this could be removed. 
 In this case is doing nothing... or maybe we're gonna manage the error in the error pipeline?


This PR is changing because in the past the error pipeline could fail and call again the error pipeline...

Reply:
nop, this is necessary since the parent class is an abstract class , so all methods defined as abstract methods have to be implemented in the subclasses